src/app/cli/src/graphql.ml

```diff
@@ -294,77 +302,65 @@ module Types = struct
     obj "FeeTransfer" ~fields:(fun _ ->
         [ field "recipient"
             ~args:Arg.[]
-            ~doc:"Receipt of fee transfer" ~typ:(non_null string)
+            ~doc:"Public key of fee transfer recipient" ~typ:(non_null string)
             ~resolve:(fun _ (pk, _) -> Stringable.public_key pk)
-        ; uint64_field "fee" ~doc:"Fee that the receipt receives"
+        ; uint64_field "fee"
+            ~doc:"Amount that the recipient is paid in this fee transfer"
             ~args:Arg.[]
             ~resolve:(fun _ (_, fee) -> Stringable.fee fee) ] )
 
-  let delegation_update =
-    obj "DelegationUpdate" ~fields:(fun _ ->
-        [ field "status"
-            ~args:Arg.[]
-            ~typ:(non_null user_command)
-            ~resolve:(fun _ user_command -> user_command)
-          (* TODO: include active field *)
-          (* TODO: include consensus field *)
-         ] )
-
   let transactions =
     let open Filtered_external_transition.Transactions in
     obj "Transactions" ~doc:"Different types of transactions in a block"
       ~fields:(fun _ ->
         [ field "userCommands"
+            ~doc:
+              "List of user commands (payments and stake delegations) \
+               included in this block"
             ~typ:(non_null @@ list @@ non_null user_command)
             ~args:Arg.[]
             ~resolve:(fun _ {user_commands; _} -> user_commands)
         ; field "feeTransfer"
+            ~doc:"List of fee transfers included in this block"
             ~typ:(non_null @@ list @@ non_null fee_transfer)
             ~args:Arg.[]
             ~resolve:(fun _ {fee_transfers; _} -> fee_transfers)
-        ; uint64_field "coinbase" ~doc:"Coinbase amount that proposer receives"
+        ; uint64_field "coinbase"
+            ~doc:"Number of tokens granted to the producer of this block"
```

Comment:
nit: to be consistent with other descriptions, would say "Amount granted to the producer of this block"

Reply:
I was moving towards "number of tokens" so as to be clear even when we add decimal points to the display of the currency.. not sure